### ISSUE TITLE
Need more detail on training own data 

### ISSUE THREAD

**Issue by yong543210** [nekitmm/starnet#34]
“What you can do, however, is you can train the net a bit more using your own data. Just prepare one or two starless images from your data and run training for 20 epochs or so. This should significantly improve results and make the net much better for your data.”

As what you wrote above. I prepare an original image and one without star, and put them in the train subfolder: original and starless. 

When I train 1 epoch, confirm there is only 1 image in the dataset. Do I need to keep or delete the sample image 2_s.tif you provide  in the dataset? Do I need to change the learning rate which is 0.002 to prevent catastrophic forgetting? Any advice is appreciated as retraining can cause unexpected result. 

Thank you very much.

**Comment by nekitmm** [comment]
Hi!

This is a very complicated topic that you just can't hope to describe in a single comment. Answering your specific questions, I'd

1. Keep the image I have as an example (optional, but I'd recommend)
2. Set the learning rate way lower and see how it reacts. I'd watch the loss changing with small learning rate and then try to increase it if nothing happens.

Overall, if you could prepare more images, that would be great.

Good luck!

**Comment by fommil** [comment]
I'm also interested in training data sets. But I'm also very interested in what training data you used in the first place, the `train` folder only has one file in it.
